src/documents/models.py

```diff
@@ -888,6 +898,8 @@ class CustomFieldInstance(models.Model):
 
     value_document_ids = models.JSONField(null=True)
 
+    value_select = models.IntegerField(null=True)
```

Comment:
This functions as an index right?  Since it doesn't need to be negative, maybe make it a `PositiveSmallIntegerField`?  I would hope ~32k entries is enough

Reply:
Sure! Yea agreed 32k enough 😆